### DISCUSSION TITLE
Hard coded limitations?

### DISCUSSION THREAD

**Discussion by michaelbrandi** [community/community#15122] [Copilot Conversations] [answered]
Is there some hard coded limitations that one should be aware of? If I wrote "// create array of 20 names", everything works as expected, but if I write "// create array of 100 names", no suggestion is ever created. Is that hard coded, or is it simply because the problem size becomes to big?

**Comment by davecheney** [comment, marked as answer]
Copilot is limited to around 1000 characters in the response. This is mostly a limitation of the OpenAI API.

**Reply by michaelbrandi** [reply]
Thank You

**Reply by christianmartini** [reply]
Is there a way to change/configure this limit?

**Reply by davecheney** [reply]
Not at the moment, but as models with a higher token count are available we revisit this constantly.

**Reply by sbond777** [reply]
@davecheney what is the prompt limit right now? It tells me "reauest filtered out" when I send a mere 300-400 token prompt.
